lib/config/validation.ts

```diff
@@ -672,3 +655,45 @@ export async function validateConfig(
   warnings.sort(sortAll);
   return { errors, warnings };
 }
+
+function validateRegexManagerFields(
+  regexManager: RegexManagerConfig,
+  currentPath: string,
+  errors: ValidationMessage[]
+): void {
+  if (is.nonEmptyArray(regexManager.matchStrings)) {
+    for (const matchString of regexManager.matchStrings) {
+      try {
+        regEx(matchString);
+      } catch (e) {
+        errors.push({
+          topic: 'Configuration Error',
+          message: `Invalid regExp for ${currentPath}: \`${String(
+            matchString
+          )}\``,
```

Comment:
```suggestion
          message: `Invalid regExp for ${currentPath}: \`${matchString}\``,
```
do we need the string function call? 🤔